lib/teaspoon/instrumentation.rb

```diff
@@ -45,14 +45,14 @@ def instrumented_response
 
     def self.ignored?(asset)
       Array(Teaspoon::Coverage.configuration.ignore).any? do |ignore|
-        asset.pathname.to_s.match(ignore)
+        asset.filename.match(ignore)
```

Comment:
```filename``` is present in Sprockets 3 & 4 and not in 2. I patched Sprockets 2 to support it in ```teaspoon/engine.rb```